aws/aws-how-to/security/index.rst

```diff
@@ -6,4 +6,5 @@ AWS provides multiple features for additional security, many of which are suppor
 .. toctree::
    :maxdepth: 1
 
+   Ubuntu Security on AWS <ubuntu-security-on-aws>
```

Comment:
Since this is not really a how-to guide, but just useful information, move this content under the 'Explanation' section.